lib/site_template/_config.yml

```diff
@@ -32,3 +32,4 @@ gems:
 exclude:
   - Gemfile
   - Gemfile.lock
+  - README.txt
```

Comment:
guys, I opted for a `.txt` file because I did not want this file to be checked into the user's git index.
Since `README` holds a special value in repositories, I'm going to rename this file as `Welcome.txt`

@crispgm you're free to create a PR that initializes a proper `README.md` with `jekyll new`. Just that there will also be this `Welcome.txt` that'll be ignored by both Git and Jekyll, if this gets merged.